entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/EntityRecordService.java

```diff
@@ -554,15 +554,32 @@ public Entity mergeEntities(Entity primary, Entity secondary) throws EuropeanaAp
     return combineEntities(primary, secondary, fieldsToCombine, true);
   }
 
-  public void updateConsolidatedVersion(EntityRecord entityRecord, Entity consolidatedEntity) {
+  public void updateConsolidatedVersionAggregatesAndDeprecation(
+      EntityRecord entityRecord, Entity consolidatedEntity) {
 
     /*
      * isAggregatedBy isn't set on Europeana Proxy, so it won't be copied to the
      * consolidatedEntity We add it separately here
      */
     Aggregation aggregation = entityRecord.getEntity().getIsAggregatedBy();
     aggregation.setModified(new Date());
+    // update the aggregates, since some proxy data can be changed
+    List<String> newAggregates = new ArrayList<String>();
+    if (aggregation.getAggregates() != null) {
+      newAggregates.add(aggregation.getAggregates().get(0));
+    }
+    if (entityRecord.getExternalProxies() != null) {
+      for (int i = 0; i < entityRecord.getExternalProxies().size(); i++) {
+        newAggregates.add(getDatasourceAggregationId(entityRecord.getEntityId(), i + 1));
+      }
+    }
+    aggregation.setAggregates(newAggregates);
     consolidatedEntity.setIsAggregatedBy(aggregation);
+
+    if (entityRecord.isDisabled()) {
+      entityRecord.setDisabled(null);
```

Comment:
this enables all organizations, but the ones that are marked for deletion in Zoho must not be enabled